php_sources/Comment.php

```diff
@@ -0,0 +1,21 @@
+<?php namespace CommentClass;
+
+class Comment {
+    protected $user;
+    protected $text;
+
+    public function __construct($user, $text) {
+        $this->user = $user;
+        $this->text = $text;
+    }
+
+    public function filterByDate($date) {
```

Comment:
Не указан тип входного параметра и возвращаемого значения.
Формирование кода не по PSR
```suggestion
public function filterByDate(Date $date): bool
{
```